examples/basic/cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,5 @@
+describe('template spec', () => {
+  it('passes', () => {
+    cy.visit('https://example.cypress.io')
```

Comment:
Generally we recommend not visiting an outside service in the tests. If the example site is down, these tests would fail. I'd recommend updating this to something that will always run independently (like the other examples).

Reply:
This is an exact copy of the default test which Cypress scaffolds, so I don't think it poses a higher risk here in the Docker repo. I used the test because I thought it might be familiar to users. (The GitHub Actions repo uses tests against https://example.cypress.io as well.)

If you don't think it is desirable to be reliant on https://example.cypress.io then I would replace the test spec using the one from https://github.com/cypress-io/cypress-docker-images/blob/master/examples/included-as-non-root-alternative/src/cypress/integration/spec.js

Let me know if you want the change done.

Reply:
The prerequisite for the test is that it should test something in a browser. It is not essential to use an external site for this, so I went ahead and converted to testing using a simple local index.html file.

Resolved in https://github.com/cypress-io/cypress-docker-images/pull/1110/commits/55a258c78e318b55a3a87696dc2aa2453a33fbeb